frontend/src/app/projects/models/diagrams/service/model-diagram.service.ts

```diff
@@ -0,0 +1,49 @@
+/*
+ * SPDX-FileCopyrightText: Copyright DB Netz AG and the capella-collab-manager contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { Observable } from 'rxjs';
+import { ModelService } from 'src/app/projects/models/service/model.service';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class ModelDiagramService {
+  constructor(private http: HttpClient, private modelService: ModelService) {}
```

Comment:
I have another opinion here: 

- The ModelService has the backendURLFactory as it is intended for the modelService itself. 
- We can reuse it here because diagrams is a subpath of the model route: If the model route changes, it just need be changed in the model service
- A new URL helper file removes the component scope - it's clearly related to the modelService (as it produces a URL to fetch models) and one file for just one function - I don't like that.

Reply:
But what we can do: Make the function static and just import it without injectable (would be the best solution)

Reply:
Change it to a static function.